### PR TITLE
Add proof of async tasks with dramatic and periodiq

### PR DESCRIPTION
**Pull request by falvaradorodriguez**
- Related with #7 

This RP is not intended to be mergable. It is therefore kept as a draft.

It is used as a proof of concept for `dramatiq` and `periodiq` in conjunction with `FastAPI`.

In the proof of concept the following is done:

- A queue is created simulating the queue that will contain the events of the contracts.
- Use `pika` to consume events from the queue and launch an asynchronous task.
- The asynchronous tasks are managed with `dramatiq`.
- An example of a cron task is also added, with `periodiq` to validate that `dramatiq` and `celery` can potentially do the same (https://dramatiq.io/motivation.html#compared-to)

It is possible to send test messages to the queue using `python scripts/send_test_message_rabbitmq.py`.

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+aio-pika==9.5.1
+dramatiq[redis, watch]==1.17.1
 fastapi[all]==0.115.4
+periodiq==0.13.0
+pika==1.3.2
```
Comment (Uxio0) [inline requirements/prod.txt:5]: Why pika and aio-pika are imported?

File: docker-compose.yml [hunk 2]
```diff
@@ -43,4 +66,8 @@ services:
       - "8888:8888"
     volumes:
       - nginx-shared:/nginx
-    command: docker/web/run_web.sh
\ No newline at end of file
+    command:
+      docker/web/wait_for_it.sh rabbitmq:5672 --timeout=30 --strict --
```
Comment (Uxio0) [inline docker-compose.yml:70]: You can use healthCheck in docker compose: https://medium.com/@saklani1408/configuring-healthcheck-in-docker-compose-3fa6439ee280

File: requirements/prod.txt (hunk 1 above)
Comment (Uxio0) [reply requirements/prod.txt:5]: Oh, I see that you are using a Blocking connection for the example, right?

File: app/main.py [hunk 3]
```diff
@@ -18,3 +21,10 @@
 api_v1_router.include_router(about.router)
 app.include_router(api_v1_router)
 app.include_router(default.router)
+
+
+@app.on_event("startup")
+async def startup():
+    loop = asyncio.get_running_loop()
+    task = loop.create_task(QueueConsumer().consume(loop))
```
Comment (Uxio0) [inline app/main.py:29]: Nice that you are using async connection

File: docker-compose.yml (hunk 2 above)
Comment (falvaradorodriguez) [reply docker-compose.yml]: Yes, bad decision. Updated [here](https://github.com/safe-global/safe-decoder-service/pull/10/commits/3612bf1cb83c7942bc9a114c0c40e18419168350)

Comment (Uxio0) [reply docker-compose.yml]: Not at all. I've just discovered you can do that a couple of months ago

**Pull request closed without merging**